lms/djangoapps/branding_by_referer/configuration_by_referer.py

```diff
@@ -0,0 +1,161 @@
+"""
+Overwites some site configuration according to the referer
+"""
+import copy
+import json
+
+from django.utils.deprecation import MiddlewareMixin
+from django.utils.six import iteritems
+from django.utils.six.moves.urllib.parse import urlparse
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangoapps.user_api.models import UserPreference
+
+
+class SetConfigurationByReferer(MiddlewareMixin):
```

Comment:
If this class is a middleware, why is it not defined in a module called middleware.py ?

Reply:
Ok, I'll place this one in the middleware.py file. In other pr I will move the BrandingByReferer middleware there as well